src/core/fbs/mutator/fetcher.ts

```diff
@@ -0,0 +1,64 @@
+import { getToken, TOKEN_USER_KEY } from "../../token";
+
+const baseURL = "https://fbs-openplatform.dbc.dk"; // use your own URL here or environment variable
+
+export const fetcher = async <ResponseType>({
+  url,
+  method,
+  headers,
+  params,
+  data
+}: {
```

Comment:
With this PR, #42 and #43 we are now at three fetchers which (should) look about the same apart from which base url and token is used.

At some point we should look into creating something that is more easily shareable.

Reply:
Agreed :-) Think that there will be a small rewrite to re-use fetchers in the next sprint when the current once are merged. I'll add it to the next meeting schedule.